packages/edition-twig/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@pattern-lab/edition-twig",
+  "version": "3.0.0-alpha.1",
+  "description": "Pattern Lab node with Twig PHP Engine",
+  "author": {
+    "name": "Evan Lovely",
+    "url": "http://evanlovely.com"
+  },
+  "maintainers": [
+    {
+      "name": "Salem Ghoweri"
+    }
+  ],
+  "main": "patternlab-config.json",
+  "scripts": {
+    "build": "patternlab build --config ./patternlab-config.json",
+    "help": "patternlab --help",
+    "install": "patternlab install --config ./patternlab-config.json",
+    "serve": "patternlab serve --config ./patternlab-config.json",
+    "start": "npm run serve",
+    "version": "patternlab --version"
+  },
+  "dependencies": {
+    "@pattern-lab/cli": "^0.0.1-alpha.19",
+    "@pattern-lab/core": "^3.0.0-alpha.13",
+    "@pattern-lab/engine-mustache": "^2.0.0-alpha.6",
+    "@pattern-lab/engine-twig-php": "^0.1.0",
+    "@pattern-lab/uikit-workshop": "^1.0.0-alpha.5",
+    "@pattern-lab/starterkit-twig-demo": "^4.0.0"
```

Comment:
Do you intend the Twig edition to contain the demo? This leads to a ton of duplicate code compared to `starterkit-twig-demo`. It's just a departure from the other editions, which have always been empty with the expectations that users install the starterkits separately. 

Truer still, the CLI will now ask for the edition and then the starterkit, so baking them both into the edition reduces the value of the CLI process.

Reply:
This PR would be a ton smaller without all these files from the starterkit, too.